xlsynth-g8r/fuzz/fuzz_targets/fuzz_ir_outline_equiv.rs

```diff
@@ -144,9 +144,9 @@ fuzz_target!(|sample: FuzzSample| {
         Ok(p) => p,
         Err(_) => return, // Early-return: infra issue constructing package
     };
-    if let Err(_e) = generate_ir_fn(sample.input_bits, sample.ops.clone(), &mut cpp_pkg, None) {
+    if let Err(_e) = generate_ir_fn(sample.ops.clone(), &mut cpp_pkg, None) {
         return; // Early-return: generator could not build function with given
-                // ops
+        // ops
```

Comment:
Maybe hoist the comment here above the return.

Reply:
done.